sql-ballerina/tests/complex-query-test.bal

```diff
@@ -238,35 +238,82 @@ type ResultDates record {
     record{| record{} value; |}? data =  checkpanic queryResult.next();
     record{}? value = data?.value;
     checkpanic dbClient.close();
 
-    string dateType = "2017-05-23+00:00";
-    string timeTypeString = "14:15:23.000+00:00";
-    string insertedTimeString = "2017-01-25T16:33:55.000+00:00";
+    string dateTypeString = "2017-05-23";
+    string timeTypeString = "14:15:23";
+    string timestampTypeString = "2017-01-25 16:33:55.0";
+    string timeWithTimezone = "16:33:55+06:30";
+    string timestampWithTimezone = "2017-01-25T16:33:55-08:00";
 
     ResultDates expected = {
-        DATE_TYPE: dateType,
+        DATE_TYPE: dateTypeString,
         TIME_TYPE: timeTypeString,
-        TIMESTAMP_TYPE: insertedTimeString,
-        DATETIME_TYPE: insertedTimeString
+        TIMESTAMP_TYPE: timestampTypeString,
+        DATETIME_TYPE: timestampTypeString,
+        TIME_TZ_TYPE: timeWithTimezone,
+        TIMESTAMP_TZ_TYPE: timestampWithTimezone
     };
-    test:assertEquals(value, expected, "Expected record did not match."); 
+    test:assertEquals(value, expected, "Expected record did not match.");
 }
 
+//type ResultDatesAsRecords record {
+//    time:Date DATE_TYPE;
+//    time:TimeOfDay TIME_TYPE;
+//    string TIMESTAMP_TYPE;
+//    time:Civil DATETIME_TYPE;
+//    string TIME_TZ_TYPE;
+//    string TIMESTAMP_TZ_TYPE;
+//};
+//
+//@test:Config {
+//    enable: false,
+//    groups: ["query", "query-complex-params"]
+//}
+//function testDateTimeRecords() {
+//    MockClient dbClient = checkpanic new (url = complexQueryDb, user = user, password = password);
+//    stream<record{}, error> queryResult = dbClient->query("SELECT date_type, time_type, timestamp_type, datetime_type"
+//       + ", time_tz_type, timestamp_tz_type from DateTimeTypes where row_id = 1", ResultDatesAsRecords);
+//    record{| record{} value; |}? data =  checkpanic queryResult.next();
+//    record{}? value = data?.value;
+//    checkpanic dbClient.close();
+//
+//    time:Date dateTypeRecord = {year: 2017, month:5, day: 23};
+//    time:TimeOfDay timeTypeRecord = {hour:14, minute:15, second:23};
+//    time:Utc timestampTypeRecord = checkpanic time:utcFromString("2017-01-25T16:33:55.000+00:00");
+//    string timestampTypeString = "2017-01-25T16:33:55.000+00:00";
+//    time:Civil dateTimeTypeRecord = {year: 2017, month:1, day: 25, hour: 16, minute: 33, second:55};
+//    string timeWithTimezone = "16:33:55.000+00:00";
+//    string timestampWithTimezone = "2017-01-25T16:33:55.000+00:00";
+//
+//    ResultDatesAsRecords expected = {
+//        DATE_TYPE: dateTypeRecord,
+//        TIME_TYPE: timeTypeRecord,
+//        TIMESTAMP_TYPE: timestampTypeString,
+//        DATETIME_TYPE: dateTimeTypeRecord,
+//        TIME_TZ_TYPE: timeWithTimezone,
+//        TIMESTAMP_TZ_TYPE: timestampWithTimezone
+//    };
+//    test:assertEquals(value, expected, "Expected record did not match.");
+//}
```

Comment:
Let's remove the commented lines. if this is due to failing test, let's create an issue to track

Reply:
removed in https://github.com/ballerina-platform/module-ballerina-sql/pull/161/commits/1a363a5654c30b1c3b4efd836eb9225713c4e5a6 no need to create an issue because that test case is newly introduced and those are tested in query-simple-params test group as well.